pkger/service.go

```diff
@@ -568,6 +569,7 @@ func ExportWithExistingResources(resources ...ResourceToClone) ExportOptFn {
 	return func(opt *ExportOpt) error {
 		for _, r := range resources {
 			if err := r.OK(); err != nil {
+				// todo: log and continue else append r to opt.Resources??
```

Comment:
does this need to be addressed in this PR?

Reply:
probably not, it was a note to investigate if that was necessary.